internal/pkg/cli/secret_init.go

```diff
@@ -0,0 +1,117 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+
+	"github.com/aws/copilot-cli/internal/pkg/config"
+)
+
+type initSecretVars struct {
```

Comment:
Nit: any reason this shouldn't be `secretInitVars`?
```suggestion
type secretInitVars struct {
```